packages/headless/src/features/generated-answer/generated-answer-slice.test.ts

```diff
@@ -292,6 +302,25 @@ describe('generated answer slice', () => {
     });
   });
 
+  describe('#updateResponseFormat', () => {
+    it('concatenates the given string with the answer previously in the state', () => {
```

Comment:
The description of this test feels nebulous to me.

What is "the given string" ? is it `'step'` ?

What is "the answer previously in the state" ? Is it `responseFormat`?

If so, how can `'step'` (a string) be concatenated with `responseFormat` (an object)?

Could we just say:

`sets the response format to the specified value`

instead?